Source/WebKit/UIProcess/API/Cocoa/WKWebView.mm

```diff
@@ -338,6 +340,58 @@ static uint32_t convertSystemLayoutDirection(NSUserInterfaceLayoutDirection dire
 
 #endif // PLATFORM(MAC)
 
+#if ENABLE(SCREEN_TIME)
+- (void)_installScreenTimeWebpageController
+{
+    if (!_page->preferences().screenTimeEnabled())
+        return;
+    if (!_screenTimeWebpageController) {
+        _screenTimeWebpageController = adoptNS([[STWebpageController alloc] init]);
+        [_screenTimeWebpageController addObserver:self forKeyPath:@"URLIsBlocked" options:(NSKeyValueObservingOptionOld | NSKeyValueObservingOptionNew) context:&screenTimeWebpageControllerBlockedKVOContext];
+
+#if PLATFORM(MAC)
+        NSView *screenTimeView = [_screenTimeWebpageController view];
+        screenTimeView.frame = self.bounds;
+        screenTimeView.translatesAutoresizingMaskIntoConstraints = NO;
+        // add to webview
```

Comment:
These are "what" comments. They're also not complete sentences. In WebKit, we usually A) prefer code that is so obvious it doesn't need comments, and B) if it really does, they should be "why" comments, in complete sentences, with capitalization and punctuation.